.github/labeler.yml

```diff
@@ -0,0 +1,97 @@
+3D:
+- changed-files:
+  - any-glob-to-any-file:
+    - /**/g3d/**
+android:
+- changed-files:
+  - any-glob-to-any-file:
+    - /**/gdx-backend-android/**
+    - /**/gdx-tests-android/**
```

Comment:
This would label changes to https://github.com/libgdx/libgdx/tree/master/tests/gdx-tests-android/assets/data as Android even if the pull requester hasn't touched Android? Since all test assets are in there. And possibly chaos if someone touches a large number of files. I like the idea.

Reply:
Hmm, good point. We could explicitly exclude this path (if I'm not mistaken): `!/**/gdx-tests-android/assets/data/**`